### PR TITLE
Add compatibility with LPR model from openvino_training_extensions

### PR DESCRIPTION
**Pull request by AlexanderDokuchaev**
- `license-plate-recognition-barrier-0001` model converted from Caffe and have two input blobs
- implementation this LPR model in [TensorFlow from openvino_training_extensions](https://github.com/opencv/openvino_training_extensions/tree/develop/tensorflow_toolkit/lpr) have one

### CODE REVIEW

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by vladimir-dudnik** [comment]
Jenkins please retry a build

**Comment by AlexanderDokuchaev** [comment]
@Wovchena 
> Please, retarget to opencv:develop

Done

> Please, updates demo's `README.md`

I dont see any reason to update readme.md.

> Please, update tests for the demo, adding `license-plate-recognition-barrier-0007`

I will no update tests for nonexistent model.

**Comment by Wovchena** [comment]
> I dont see any reason to update readme.md.

The demo works with `license-plate-recognition-barrier-0007` thanks to this PR.

> I will no update tests for nonexistent model.

This is the existent model: https://github.com/opencv/open_model_zoo/blob/develop/models/public/license-plate-recognition-barrier-0007/model.yml. According to `source` field in this yml, this is the model you refer to in the description of this PR.

File: demos/security_barrier_camera_demo/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ of the detection results. You can use a set of the following pre-trained models
 * `vehicle-license-plate-detection-barrier-0106`, which is a primary detection network to find the vehicles and license plates
 * `vehicle-attributes-recognition-barrier-0039`, which is executed on top of the results from the first network and
 reports general vehicle attributes, for example, vehicle type (car/van/bus/track) and color
-* `license-plate-recognition-barrier-0001`, which is executed on top of the results from the first network
+* `license-plate-recognition-barrier-0001` and `license-plate-recognition-barrier-0007`, which is executed on top of the results from the first network
```
Comment (IRDonch) [inline demos/security_barrier_camera_demo/README.md]:
```suggestion
* `license-plate-recognition-barrier-0001` or `license-plate-recognition-barrier-0007`, which is executed on top of the results from the first network
```

**Pull request merged**